CIP-0989/README.md

```diff
@@ -0,0 +1,131 @@
+
+Ultimately, a DID is an attestation conferred upon a user from some other known and trusted entity/entities.
+
+The existing PRISM protocol enables the use of Decentralized Identifiers (DIDs) and Verifiable Credentials (VC).
+
+DID's are attached to a specific end-user public key, and verifiable via issuing a signature from the corresponding private key, after checking the DID status on PRISM.
+
+Thus, end-users may confirm ownership of a DID by providing an EdDSA signature with the correct private key. 
+
+DIDs are binary data encoded as hexadecimal and Base64, as an example:
+
+`(did:prism:7e5b03ee503c6d63ca814c64b1e6affd3e16d5eafb086ac667ae63b44ea23f2b:CmAKXhJcCgdtYXN0ZXIwEAFCTwoJc2VjcDI1NmsxEiA4BHOH6oJYfe_pVSqXl69HcRW304Mk4ExOwJye9VWcGBogOBL32BWDzjqnSqoQss8x_qv_02fwyrEL13QCKT9KRoQ=,207)`
+
+### Binding of DID & Stake Delegation
+We require a mechanism to bind an end-user's DID to the stake delegation destined for an ISPO.
+
+We can leverage the existing staking mechanism. During the process of delegating stake, an end-user will register a _delegation certificate_, this certificate permits a given SPO to include said end-user's ADA as part of their stake distribution.
+
+A DID may be submitted, as encrypted meta-data in the **same transaction** that delivers the underlying delegation certificate. Critically the transaction body is signed by both the spending private/secret key and the staking private/secret key. This has a number of serendipitous consequences:
+
+* The SPO hosting the ISPO can verify the end-user has ownership of the secret staking key required to validly delegate stake to the SPO.
+* The SPO hosting the ISPO can verify the end-user has ownership of the secret spending key required to sign the transaction which itself includes the DID as a transaction metadata field.
+* We can use the end-users staking key in an ECDH (Elliptic Curve Diffie-Hellman) interaction to generate shared symmetric key material.
+
+This approach essentially cryptographically binds a DID to a stake delegation.
+
+### DID Privacy
+In order to submit a valid DID for a given ISPO event, an end-user must include a DID in the transaction metadata which is being submitted to register stake.
+This DID information should be confidential and only visible to the concerned parties (end-user and SPO). Thus we propose a mechanism for cryptographic key agreement between end-user and SPO in a non-interaction fashion, the resulting key material can be used to protect the DID data.
+
+We will leverage `static:static ECDH` (Elliptic curve Diffie-Hellman) in order to agree a symmetric key, this process will leverage the end-user's existing staking key pair, and require a **new hot key pair** on the SPO side, known as the *DID decryption key pair*. This process assumes the same curve and curve generator point are being used by both parties.
+
+
+#### Key Agreement Process
+$$
```

Comment:
This won't render properly on markdown.

Reply:
I tried to make this work on GitHub with a simple change to the TeX formatted block & couldn't.  @johnalanwoods the math terms just need to be broken out from the text with single `$` delimiters.